src/main/kotlin/com/team4099/robot2020/subsystems/LED.kt

```diff
@@ -0,0 +1,67 @@
+package com.team4099.robot2020.subsystems
+
+import com.team4099.lib.logging.HelixEvents
+import com.team4099.lib.logging.HelixLogger
+import com.team4099.lib.subsystem.Subsystem
+import com.team4099.robot2020.config.Constants.LED.Color
+import com.team4099.robot2020.config.Constants
+import edu.wpi.first.wpilibj.AddressableLED
+import edu.wpi.first.wpilibj.AddressableLEDBuffer
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard
+
+object LED : Subsystem {
+
+    // PWM port 9
```

Comment:
these comments are kinda bad because they incorrectly explain things/are in the wrong place